### DISCUSSION TITLE
How do we solve Maui bugs ourselves? Step-by-Step, what do we do with the Maui repository?

### DISCUSSION THREAD

**Discussion by jonmdev** [dotnet/maui#21219] [Ideas] [answered]
How do we solve Maui bugs? I posted a question with an example of something I want to fix here:

https://github.com/dotnet/maui/issues/21201

I can download the Maui repo using Github but I have no idea what to do with it after. I saw some demo projects for bug fixes were hidden in there. But I don't know how to add a new one and run it.

What I would like to do is add a demo project to the Maui repository that I can build and run right from that combined project so I can debug and test things inside the Maui code for solutions.

Then if I find a solution, I can push the update with the demo project and correction included.

Is this how it is supposed to be done?

Thanks for any help. Please be clear about what the steps we are supposed to take. I can easily make bug reports with demo projects showing the problems. I have submitted at least 30 bug reports, most of which I sent 5-6 months ago, each with <100 lines code to prove the problems. 

But I am not sure how I am supposed to debug and fix Maui directly for the ones that are otherwise not being fixed.

Thanks for any help.

**Comment by jsuarezruiz** [comment, marked as answer]
I have a video where I try to explain all the steps and details https://www.youtube.com/watch?v=8LRfy6Omp9I, I hope it helps you.
